### ISSUE TITLE
Update documentation

### ISSUE THREAD

**Issue by asiddi24** [malmans2/oceanspy#241]
* OceanSpy version:0.2.0

### Description

Oceanspy's documentation is due for an update for some time now. Here are some important action items on that.

- [ ] Update tutorials for accessing ECCO and Poseidon datasets
- [x] Update api for `llc_rearrange`
- [ ] Update tutorials for heat and salt budget calculations using the ASR runs and or ECCO data
- [ ] We should update the docs using the [Executable book project](https://executablebooks.org/en/latest/) from Jupyter.
- [x] Update developers list with @MaceKuailv 
- [x] Other minor fixes

Hoping that this can be done so that we can release v0.2.1 ?

**Comment by ThomasHaine** [comment]
Agreed. The 3rd items will also address #71. Please go ahead working on the list and I'll check the updated docs.

**Comment by ThomasHaine** [comment]
Also the 5th item closes #240 .
